cinemascience/cinema_debye_scherrer
# Switch main.html -> index.html + add docker nginx instructions

Work included:
  - Switches `main.html` -> `index.html`
    - Making this change because `index.html` is the default homepage content filename for websites. This allows not having to navigate to `http://localhost:8000/main.html` but instead can directly use `http://localhost:8000/`. 
    - Quick link I found where it explains that `index.html` is this default homepage filename: https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/Dealing_with_files#what_structure_should_your_website_have

  - Update README for the `main.html` -> `index.html` switch

  - Added a small subsection to README under the "Loading the viewer" for running Cinema:Debye_Cherrer in a nginx docker container
    - See this section of my branch for the new content: https://github.com/marshallmcdonnell/cinema_debye_scherrer/tree/main-html-to-index-html#using-docker

To test:
  - For the `main.html` -> `index.html`, use the original instructions to ensure it still works.
    - https://github.com/cinemascience/cinema_debye_scherrer#simple-tutorial
  
  - For running from a webserver, you now should be able to drop the `main.html` from the URL, using these instructions:
    - https://github.com/cinemascience/cinema_debye_scherrer#loading-the-viewer

  - Check README for modifications that reads correctly

  - For new docker, will need docker installed. Then, to test, just follow the new README instructions:
    - https://github.com/marshallmcdonnell/cinema_debye_scherrer/tree/main-html-to-index-html#using-docker
